ext/opcache/jit/zend_jit_ir.c

```diff
@@ -12786,6 +12787,18 @@ static int zend_jit_fetch_dim_read(zend_jit_ctx       *jit,
 				}
 			}
 
+			if (opline->opcode == ZEND_FETCH_LIST_R) {
+				ir_ref ref;
+
+				may_throw = 1;
+				if ((op1_info & MAY_BE_UNDEF) || (op2_info & MAY_BE_UNDEF)) {
+					ref = jit_ZVAL_ADDR(jit, orig_op1_addr);
+				} else {
+					jit_SET_EX_OPLINE(jit, opline);
+					ref = jit_ZVAL_ADDR(jit, op1_addr);
+				}
+				ir_CALL_1(IR_VOID, ir_CONST_FC_FUNC(zend_jit_invalid_array_use), ref);
+			}
```

Comment:
Nit: possibly merge this with the block bellow, as it's the same code except for the call :)

Reply:
Done :)